src/packs.c

```diff
@@ -199,7 +240,51 @@ int download_subscribed_packs(struct list *subs, struct manifest *mom, bool requ
 			}
 		}
 	}
-	progress_report(list_length, list_length); /* Force out 100% */
 	info("\n");
+	list_free_list(need_download);
+
 	return swupd_curl_parallel_download_end(download_handle, NULL);
 }
+
+double packs_query_total_download_size(struct list *subs, struct manifest *mom)
```

Comment:
this functions should be static and not exported

Reply:
done